modules/schema/src/main/java/org/apache/ignite/internal/schema/SchemaDescriptor.java

```diff
@@ -173,6 +177,11 @@ public void validateColumnIndex(int colIdx) {
         Objects.checkIndex(colIdx, length());
     }
 
+    /** Returns true if physical order matches the logical order, return false otherwise. */
```

Comment:
If you are going to fix the map capacity problem mentioned by zstan, then also change this doc a little :sweat_smile: 

```suggestion
    /** Returns true if physical order matches the logical order, false otherwise. */
```

Reply:
done